pom.xml

```diff
@@ -117,7 +110,6 @@
       <plugins>
         <plugin>
           <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
Remove this section entirely. You need to be Maven 2 to use the new parent pom, so no need for enforcing what remains